src/store/RelayStore.js

```diff
@@ -171,11 +176,35 @@ var RelayStore = {
     );
   },
 
+  /**
+   * Adds an update to the store and commits it immediately. Returns
+   * The RelayMutationTransaction.
+   */
+  commitUpdate(
+    mutation: RelayMutation,
+    callbacks?: RelayMutationTransactionCommitCallbacks
+  ): RelayMutationTransaction {
+    const transaction = this.applyUpdate(mutation, callbacks);
+    transaction.commit();
+    return transaction;
+  },
+
+  /**
+   * @deprecated
+   *
+   * Method renamed to commitUpdate
+   */
   update(
     mutation: RelayMutation,
     callbacks?: RelayMutationTransactionCommitCallbacks
   ): void {
-    this.applyUpdate(mutation, callbacks).commit();
+    warning(
+      false,
+      'RelayStore.%s is deprecated. Please use RelayStore.%s instead.',
```

Comment:
No need for the `%s` placeholders if you have static strings.

The message string has to be static so we can easily group warnings by it; only dynamic parts have to be added with placeholders.